cmd/ui/src/components/GraphEvents.tsx

```diff
@@ -157,6 +157,8 @@ export const GraphEvents: FC<GraphEventProps> = ({
                     newData.highlighted = true;
                 }
 
+                newData.forceLabel = true;
```

Comment:
Would it be possible to set this forceLabel property for both nodes and edges during the graph initialization in GraphView.tsx instead of updating the property every time the reducer runs?